common/Exception.h

```diff
@@ -58,15 +59,16 @@ class Exception final {
     }
     template <typename... TArgs>
     [[noreturn]] static inline bool enforce_handler(std::string_view check, std::string_view file, int line,
-                                                    std::string_view message, const TArgs &...args)
+                                                    fmt::string_view message, const TArgs &...args)
         __attribute__((noreturn)) {
-        raise("{}:{} enforced condition {} has failed: {}", file, line, check, fmt::format(message, args...));
+        raise("{}:{} enforced condition {} has failed: {}", file, line, check,
+              fmt::vformat(message, fmt::make_format_args(args...)));
     }
 };
 
-template <typename... TArgs> [[noreturn]] bool Exception::raise(const TArgs &...args) {
+template <typename... TArgs> [[noreturn]] bool Exception::raise(fmt::string_view formatString, const TArgs &...args) {
     Exception::failInFuzzer();
-    std::string message = fmt::format(args...);
+    std::string message = fmt::vformat(formatString, fmt::make_format_args(args...));
```

Comment:
I want to try this again to see if I can get this to type check 🤔 

I was having some problems getting this to type check and had to go back to the runtime-checked formatting ones, instead of the constexpr ones.